dependency-check-maven-config.xml

```diff
@@ -84,4 +128,103 @@
    ]]></notes>
         <cve>CVE-2014-0003</cve>
     </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        this CVE is generating a lot of false positives it should only include jackson-dataformat-xml jar
+         but it is catching all jackson dependencies unfortunately I can't fix it so if we start depending
+          on a vulnerable version of jackson-dataformat-xml this will sill suppress it.
+   ]]></notes>
+        <cve>CVE-2016-3720</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+   file name: commons-beanutils-1.8.3.jar
+   shiro-core has a dependency on this but it doesn’t expose commons-beanutils to user input so it wouldn't pose a risk like the struts library that is called out in the CVE
+   ]]></notes>
+        <sha1>686EF3410BCF4AB8CE7FD0B899E832AABA5FACF7</sha1>
```

Comment:
Is this the has of `shiro-core`? If so and we upgrade `shiro-core` then I assume this vulnerability will re-present itself, correct?